### PR TITLE
[5.1-Backports] Correctly handle bug titles with double quote in new update form

### LINKED ISSUE #3714: Cannot edit builds/bugs in an update
After update to bodhi-5.0.0, I cannot edit builds nor bugs for an update.

- I haven't found any way of editing it in GUI (when clicking "edit", there is a list of builds and a list of bugs, but it cannot be edited or at least it's not clear how).
- In CLI, I get "Password:" prompt even though I have valid kerberos ticket.

### PR DESCRIPTION
**Pull request by mattiaverga**
Double quotes in bug titles need to be escaped with `\\` to be compatible with JSON.parse().
Fix #3714 #3814 

Signed-off-by: Mattia Verga <mattia.verga@tiscali.it>

### CODE REVIEW

File: bodhi/server/util.py [hunk 1]
```diff
@@ -1303,3 +1303,14 @@ def pyfile_to_module(
         e.strerror = 'Unable to load file (%s)' % e.strerror
         raise
     return d
+
+
+def json_escape(text: str) -> str:
+    """Escape double quotes for JSON.parse compatibility.
+
+    Args:
+        text (str): The text to sanitize.
```
Comment (sebwoj) [inline bodhi/server/util.py]: Since you are using type annotations, let's drop 'str' here.

File: bodhi/server/util.py [hunk 2]
```diff
@@ -1303,3 +1303,14 @@ def pyfile_to_module(
...
+    Returns:
+        str: Escaped text.
```
Comment (sebwoj) [inline bodhi/server/util.py]: Same here.

**Review by sebwoj — CHANGES_REQUESTED** [review]

File: bodhi/server/util.py (hunk 1 above)
Comment (mattiaverga) [reply bodhi/server/util.py]: corrected, thanks

File: bodhi/server/util.py (hunk 2 above)
Comment (mattiaverga) [reply bodhi/server/util.py]: corrected

**Review by ryanlerch — APPROVED** [review]

**Pull request merged**